doc/man/8/rbd.rst

```diff
@@ -361,6 +361,11 @@ Commands
 :command:`bench-write` [--io-size *size-in-B/K/M/G/T*] [--io-threads *num-ios-in-flight*] [--io-total *total-size-to-write-in-B/K/M/G/T*] [--io-pattern seq | rand] *image-spec*
```

Comment:
Agree with Jason -- I would better to remove this entry, so new users even not suspect it exists. Though marking it as deprecated here is fine to me too.

I would even like to see it was removed from mini help too and to output the "deprecated" warning when it is run with this option, but you need to modify Shell::print_help for this.